src/ui/activity-iframe-view.js

```diff
@@ -124,28 +125,29 @@ export class ActivityIframeView extends View {
    * @return {!Promise}
    */
   onOpenIframeResponse_(port, dialog) {
-    this.port_ = port;
+    this.portResolved_ = port;
```

Comment:
I'd rather call this port_ and the method portPromise_()

Reply:
there's already a variable called portPromise_ on Line 91